packages/c/chartjs-plugin-annotation.json

```
@@ -14,13 +14,13 @@
     "url": "git://github.com/chartjs/chartjs-plugin-annotation.git"
   },
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/chartjs/chartjs-plugin-annotation.git",
+    "source": "npm",
+    "target": "chartjs-plugin-annotation",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "dist",
         "files": [
-          "chartjs-plugin-annotation*"
+          "**/*"
```

```suggestion
          "*.js"
```
A globstar is dangerous as it can lead to cdnjs tracking unintended files, let's use a more conservation glob that only matches [the files we need](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.js&matches=false&tests=%2F%2F%20Imported%20from%20chartjs-plugin-annotation%20NPM%20package&tests=dist%2Fchartjs-plugin-annotation.esm.js&tests=dist%2Fchartjs-plugin-annotation.js&tests=dist%2Fchartjs-plugin-annotation.min.js&tests=package.json&tests=LICENSE.md&tests=README.md&tests=types%2Felement.d.ts&tests=types%2Fevents.d.ts&tests=types%2Findex.d.ts&tests=types%2Flabel.d.ts&tests=types%2Foptions.d.ts).